### PR TITLE
Pull request for HTML/CSS Project

### PR DESCRIPTION
**Pull request by GabrielJSuarez**
Pull request to merge page for the project Backgrounds And Gradients

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                    </div>
+
+                    <div class="navbar">
+                        <a class="" href="#">Mac</a>
+                        <a class="" href="#">iPod</a>
+                        <a class="" href="#">iPhone</a>
+                        <a class="" href="#">iPad</a>
+                        <a class="" href="#">iTunes</a>
+                        <a class="" href="#">Support</a>
+                    </div>
```
Comment (expjazz) [inline index.html]: - [ ] There should be 8 items + the search bar. You missed the store.

- [ ] Also the layout of the first and last elements is different (they all have the same width). Kindly address that.

| Yours | Original |
|:---:|:---:|
|image |image |

**Review by expjazz — CHANGES_REQUESTED** [review]
Status: Changes Required :recycle: 

#### You did a great job on this page. It looks almost perfect. Just one small adjustment in the navbar:

- [ ] image

:arrow_up: 
You only have 8 (check the inline comments below for more detail).

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

File: style.css [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+/* navigation bar */
+
+.nav-container {
+    border: 1px solid #646764;
+    border-radius: 0.3rem;
+    width: 75vw;
+    margin: 5rem auto 0;
+    background: linear-gradient(0deg, #6c6c6c, #505050 50%, #646764 50%, #848484 100%);
+}
```
Comment (kitpao) [inline style.css]: - [ ] The first and last elements in your navbar are bigger than the rest, you can fix the issue just by giving a 'fixed' width to it. **Hint:** use rem units and see the difference :) 
Your navbar:
image

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hello @GabrielJSuarez 

You did nicely adding the missing store element, and besides that, you really did a great job with the rest of the elements 🏅 , you won't have major problems in the next projects 😉 . Now everything that remains is the size of your first and last elements. Kindly check below to see a hint ⬇️ 

### ♻️ Status: Your project requires 1 additional change ♻️
➡️ Please request another review once you are ready 📝 
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Review by kitpao — APPROVED** [review]
# 👋 Hi again @GabrielJSuarez

Congratulations! Your page is completed and looks great 👏 

- [ ] **OPTIONAL** Now that everything is fine, you can update your screenshot so it displays your finished page 🌟 

### 👏 Your project has been approved 👏
### 🚀 You may merge to master and submit the project completition form 🚀

___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Pull request merged**